src/convertToObject.js

```diff
@@ -12,7 +13,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssNormalized = {};
+
+  sourceString.split(';').forEach((element) => {
```

Comment:
try to rewrite according to this requirement:

![image](https://github.com/mate-academy/js_style-to-object/assets/91946036/c5c326fe-22c4-4cfa-8609-e4addc578807)